### ISSUE TITLE
jspm install @types/react-router@2.0.33: permission denied [mac, linux]

### ISSUE THREAD

**Issue by tamird** [jspm/jspm-cli#2059]
`jspm install @types/react-router@2.0.33`

```
err  Error on download for npm:@types/react-router
Error: EACCES: permission denied, open '/root/.jspm/packages/npm/@types/react-router@2.0.33/lib/History.d.ts'
    at Error (native)
```

Tested with 0.16.45 and 0.17.0-beta.28; both produce this error.

@guybedford

**Comment by guybedford** [comment]
Looking into this, but please do help clean up the github auth mess that has been created.

**Comment by guybedford** [comment]
Fixed added for 0.17 in https://github.com/jspm/npm/commit/c5d994b55f5b0d72c1e66e35a501bd655941efb8 pending release.

**Comment by tamird** [comment]
So...upstream is mucking up permissions when they package up the release?
Is this what `npm` does?

**Comment by guybedford** [comment]
Released in jspm-npm 0.29.5.
